benefits/tech_scheme.md

```diff
@@ -0,0 +1,19 @@
+# Tech-scheme
```

Comment:
Can we call this benefit something a little more obvious? "Help to buy tech", "Help to buy personal tech", "Financial help for buying tech", or something like that. The file should then be renamed to match.

Reply:
How about 'Techscheme - helping you to buy the latest tech' ?

Reply:
This about it in terms of a link in the benefits section of the README. "Techscheme" doesn't add much value to the reader.

Reply:
Ok didn't want to change to far from actual name and then cause confusion later on :D 
So shall we agree on 'Help to buy Tech' ?